File: P412 Fizz Buzz/Solution.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> output(n);
        for(int i=0;i<n;i++) {
            if((i+1)%3==0 && (i+1)%5==0) {
                output[i] = "FizzBuzz";
            }
            else if((i+1)%5==0) {
                output[i] = "Buzz";
            }
			else if((i+1)%3==0) {
                output[i] = "Fizz";
            }
            else {
                output[i] = to_string(i+1);
            }
        }
        return output;
    }
};

File: P412 Fizz Buzz/Solution.py
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        list = []
        for i in range(1, n+1):
            if i%15 == 0:
                list.append("FizzBuzz")
            elif i%5 == 0:
                list.append("Buzz")
            elif i%3 ==0:
                list.append("Fizz")
            else:
                list.append(str(i))
        return list